packages/website/releases/1.4.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Release 1.4
+slug: 1.4
+authors:
+  - name: Rebecca Bol
+    title: Software Engineer | Data Visualization
+    url: https://github.com/reb-dev
+    image_url: https://avatars.githubusercontent.com/u/52078477
+image: https://unovis.dev/img/unovis-social.png
+hide_table_of_contents: false
+date: 2024-04-02T10:00
+---
+
+Version `1.4.0` of _Unovis_ is finally here! This update is packed full with enhancements including
+the new Annotations component, expanded Graph features, and a number of bug fixes to improve overall
+stability of the library.
+
+## Release Highlights
+### 📝 New component: Annotations
+Introducing _Annotations_, a versatile new component that enables you to overlay customized, stylized text on top of your visualizations. Whether you want to highlight points of interest, annotate trends, or simply add text labels to your charts or graphs, _Annotations_ is designed to integrate smoothly with any Unovis component. 
+
+Check out _Annotations_'s [documentation](https://unovis.dev/docs/auxiliary/Annotations) and [gallery example](https://unovis.dev/gallery/view?collection=Auxiliary%20Components&title=Chart%20Annotations) to learn how to use it.
+
+<img alt="annotations" src="https://github.com/f5/unovis/assets/755708/e2d63880-5e65-406f-b7de-9a5f893b8608"/>
+
+### 🔗 Graph Link Curvature
```

Comment:
I would mention SVG icons in the title too, because for some users it can be a big deal.